src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs

```diff
@@ -372,5 +372,67 @@ public async Task Listener_AwaitsConnection_ListenerSurvivesGC()
                     await connection.DisposeAsync();
                 }).WaitAsync(TimeSpan.FromSeconds(5)));
         }
+
+        [Fact]
+        public async Task Listener_AlpnChange() {
```

Comment:
Maybe,
```suggestion
        public async Task Listener_AlpnNarrowingDown() {
```
? Otherwise, it feels like it is a test for changing ALPN while running